scripts.d/support/install-setup-ng

```diff
@@ -4,6 +4,41 @@ source $(dirname ${BASH_SOURCE[0]})/install-setup-common
 
 AUTO=0
 
+function desktop-icon()
```

Comment:
And removed from here to, and so using the one in `install-setup-common`. The whole point of the new file is to avoid code duplication, so please don't reintroduce some.